jl4-core/src/L4/Lexer.hs

```diff
@@ -855,129 +914,38 @@ displayPosToken :: PosToken -> Text
 displayPosToken (MkPosToken _r tt) =
   displayTokenType tt
 
+inversedCompleteLookup :: (Eq a, Show a) => a -> Map c a -> c
+inversedCompleteLookup k = fromJust . (\case {Nothing -> error ("error while looking up: " <> show k); a -> a}) . lookup k . map (\(a, b) -> (b, a)) . Map.toList
```

Comment:
:scream: 

more line breaks, more comments, more info in the impossible `error` case, please!

Reply:
oh also this is a leftover, it has fromJust already :P